src/webgpu/util/floating_point.ts

```diff
@@ -708,6 +709,39 @@ export abstract class FPTraits {
     return new FPInterval(this.kind, n, n);
   }
 
+  /**
+   * WGSL specifies unbounded precision:
+   *   https://www.w3.org/TR/WGSL/#floating-point-accuracy
+   * In most computations doubles (number in js) can provide enough
+   * precision to give the correctly rounded interval.
```

Comment:
nit: Maybe "In most computations doubles (number in js) are equivalent to for correctly rounded intervals."